docs/security/bot-protection.mdx

```diff
@@ -14,5 +14,4 @@ To protect your sign-ups from bots, Clerk leverages data from our CDN to determi
 
 ## Limitations
 
-- Clerk's Expo SDK does not support **Bot sign-up protection** yet.
```

Comment:
Maybe we should mention here that if you are building a native app (chrome-extension, expo, ios, etc) you need to enable the Native API.
WDYT?

Reply:
Agreed!